buildsystem/spack/ascent/spack.yaml

```
@@ -0,0 +1,96 @@
+spack:
+  specs:
+  - patch
+  - exago@develop+cuda cuda_arch=70
+  view: false
+  concretizer:
+    unify: when_possible
+    reuse: true
```

```suggestion
    reuse: false
```

@nkoukpaizan you'll want to do this to get spack to build it's own Python I think. Worst case scenario is that you'll also have to  `spack concretize` && `spack uninstall python` in order for it to not re-use it.

---

Added and triggered a rebuild in my latest commit